DataHub/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,219 @@
+
+   > [Info] If you want to change the attribute set assignments of the selected attribute, you have to modify the corresponding attribute set.
+
+4. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Edit attribute* view is closed.  
+
+
+
+## Deactivate an attribute
+
+It is recommended to deactivate an attribute instead of deleting it to prevent any problems because of existing dependencies. If you deactivate an attribute, it is no longer available for new attribute sets and it is hidden in existing products with this attribute.   
+Deactivated attributes are not included in the completeness calculation even if they are required.
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Click the attribute you want to deactivate in the list of attributes.   
+  The *Edit attribute* view is displayed. The *Data* tab is preselected.
+
+  ![Edit attribute data](../../Assets/Screenshots/DataHub/Settings/Attributes/EditAttribute_Data.png "[Edit attribute data]")
+
+2. Disable the *Active* toggle in the *Data* tab.
+
+   > [Info] If you want to reactivate an attribute, enable the *Active* toggle.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The attribute has been deactivated. The *Edit attribute* view is closed.
+
+4. Press **F5** to initialize the *Core1 Platform* and to apply the changes.   
+
+
+
+## Delete an attribute
+
+You can move an attribute to the *DELETED ATTRIBUTES* tab if it is no longer needed.
+As there are usually dependencies on an attribute, for example through an attribute set or created products, it is strongly recommended not to delete an attribute.
+Instead, you can deactivate an attribute and thus prevent its use, see [Deactivate an attribute](#deactivate-an-attribute).
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+> [Info] By default, a certain number of attributes has been predefined in the *PIM Basic Set* attribute set.
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to delete in the list of attributes.   
+  The editing toolbar is displayed above the attributes list.
+
+2. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar. The *Confirm deletion of attribute?* window is displayed.
+
+  > [Info] In the window all dependencies to attribute sets, ETL-mappings and entity entries are displayed. Be aware that problems may occur if you delete an attribute with existing dependencies.
```

Comment:
```suggestion
  > [Info] In this window, all dependencies to attribute sets, ETL-mappings and entity entries are displayed. Be aware that problems may occur if you delete an attribute with existing dependencies.
```
Alternativ:  All dependencies to attribute sets, ETL-mappings and entity entries are displayed in this window. (Auch im PIM? -> vereinheitlichen)